src/Hooks.jsx

```diff
@@ -0,0 +1,18 @@
+// src/useLocalStorage.js
+
```

Comment:
`hooks` should be a separate folder inside `src`.
 Hooks `useSemiPersistentState.jsx` or `useLocalStorage.jsx` should be inside the folder `hooks`. 

But I don't see useLocalStorage.jsx hook used anywhere. Could you delete it please?